client/src/components/AppHeader/AppHeader.js

```diff
@@ -1,12 +1,11 @@
-import logo from './logo.png';
 import React from 'react';
-import './style.scss'
+
+import '../../styles/index.scss';
```

Comment:
I think the index is not the best name for a file that does something, maybe "style.scss"

Reply:
I'm also used to such a name, but it was given by the skeleton developer, so if he does not mind we can rename it.